ale/formatters/usgscsm_formatter.py

```diff
@@ -0,0 +1,91 @@
+import json
+from ale.base.type_sensor import LineScanner, Framer
+
+def to_usgscsm(driver):
+    isd_data = {}
+
+    # exterior orientation
+    body_radii = driver.target_body_radii
+    isd_data['radii'] = {
+        'semimajor' : body_radii[0],
+        'semiminor' : body_radii[1],
+        'unit' : 'm'
+    }
+    positions, velocities, position_times = driver.positions
+    isd_data['sensor_position'] = {
+        'positions' : positions,
+        'velocities' : velocities,
+        'unit' : 'm'
+    }
+    sun_positions, sun_velocities, _ = driver.sun_positions
+    isd_data['sun_position'] = {
+        'positions' : sun_positions,
+        'velocities' : sun_velocities,
+        'unit' : 'm'
+    }
+    rotation_chain = driver.rotation_chain
+    sensor_to_target = rotation_chain.rotation(rotation_chain[-1], rotation_chain[0])
```

Comment:
This doesn't match the vispy transformation chain, but it's easier to mock in for testing purposes. I'd like the rotation interface to work on frame nodes and then you request the rotation to another node, but that's rather complicated to mock in for testing right now.